algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -1332,5 +1332,10 @@ object CMSHasherImplicits {
       CMSHasher.hashBytes(a, b, width)(x.getBytes("UTF-8"))
   }
 
+  implicit object CMSHasherBigDecimal extends CMSHasher[BigDecimal] {
```

Comment:
I went a little too far in the "mechanised patch" thing. Should've had paid more attention. this and the two previous fixed.

Shouldn't we gut CMSHasherImplicits, move its contents to CMSHasher's companion, leave "implicit vals" in place for compatibility, and @deprecate the whole thing?